WalletWasabi.Fluent/ViewModels/Wallets/Send/PrivacySuggestionsFlyoutViewModel.cs

```diff
@@ -58,8 +58,13 @@ public async Task BuildPrivacySuggestionsAsync(Wallet wallet, TransactionInfo in
 
 		if (hasChange && !isFixedAmount && !info.IsPayJoin)
 		{
-			var suggestions =
-				ChangeAvoidanceSuggestionViewModel.GenerateSuggestionsAsync(info, destination, wallet, linkedCts.Token);
+			int originalInputCount = transaction.SpentCoins.Count();
+
+			// Use at most 5 coins.
+			int maxInputCount = (int)(Math.Max(3, originalInputCount * 1.3));
```

Comment:
`Math.Max(3, originalInputCount * 1.3)` this won't use at most 5 coins. It will use many more, especially if you send more than half of your balance.

I hope it was intended, and only the comment is wrong.

Reply:
Well, for example: 
Target was 100$ `~0.0025 9483 BTC`
Balance was 150$.
`originalInputCount` was `18` , `maxInputCount` became `23`, but Bnb only chose `8-10` coins.

Another example:
Target 111$ `~0.0028 8026 BTC`
Balance 1079$
`originalInputCount` was `14` , `maxInputCount` became `18`, but Bnb only chose `8 and 7` coins for the suggestions.

So yeah, the number of coins are not larger than the allowed maximum. But... wasn't the issue about those suggestion which used 8-10 coins?

Reply:
Yes, you are right. I tested it a bit further, and it makes sense now.